spec.html

```diff
@@ -16261,179 +16261,49 @@ <h2>Syntax</h2>
-    <p>The ECMAScript white space code points are listed in <emu-xref href="#table-white-space-code-points"></emu-xref>.</p>
-    <emu-table id="table-white-space-code-points" caption="White Space Code Points" oldids="table-32">
-      <table>
-        <tr>
-          <th>
-            Code Points
-          </th>
-          <th>
-            Name
-          </th>
-          <th>
-            Abbreviation
-          </th>
-        </tr>
-        <tr>
-          <td>
-            `U+0009`
-          </td>
-          <td>
-            CHARACTER TABULATION
-          </td>
-          <td>
-            &lt;TAB>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+000B`
-          </td>
-          <td>
-            LINE TABULATION
-          </td>
-          <td>
-            &lt;VT>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+000C`
-          </td>
-          <td>
-            FORM FEED (FF)
-          </td>
-          <td>
-            &lt;FF>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+FEFF`
-          </td>
-          <td>
-            ZERO WIDTH NO-BREAK SPACE
-          </td>
-          <td>
-            &lt;ZWNBSP>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            any code point in general category “Space_Separator”
-          </td>
-          <td>
-          </td>
-          <td>
-            &lt;USP>
-          </td>
-        </tr>
-      </table>
-    </emu-table>
-    <emu-note>
-      <p>U+0020 (SPACE) and U+00A0 (NO-BREAK SPACE) code points are part of &lt;USP>.</p>
-    </emu-note>
-    <emu-note>
-      <p>Other than for the code points listed in <emu-xref href="#table-white-space-code-points"></emu-xref>, ECMAScript |WhiteSpace| intentionally excludes all code points that have the Unicode “White_Space” property but which are not classified in general category “Space_Separator” (“Zs”).</p>
-    </emu-note>
     <h2>Syntax</h2>
     <emu-grammar type="definition">
       WhiteSpace ::
-        &lt;TAB&gt;
-        &lt;VT&gt;
-        &lt;FF&gt;
-        &lt;ZWNBSP&gt;
-        &lt;USP&gt;
+        &lt;U+0009 (CHARACTER TABULATION)&gt;
+        &lt;U+000B (LINE TABULATION)&gt;
+        &lt;U+000C (FORM FEED)&gt;
+        &lt;U+FEFF (ZERO WIDTH NO-BREAK SPACE)&gt;
+        &gt; any code point with the Unicode General_Category &ldquo;Space_Separator&rdquo;
     </emu-grammar>
+    <emu-note>
+      <p>U+0020 (SPACE) and U+00A0 (NO-BREAK SPACE) are matched by |WhiteSpace| as the both have a Unicode General_Category of “Space_Separator”.</p>
```

Comment:
```suggestion
      <p>U+0020 (SPACE) and U+00A0 (NO-BREAK SPACE) are matched by |WhiteSpace| as both have a Unicode General_Category of “Space_Separator”.</p>
```
(or change "the" to "they")